packages/bridging/src/types.ts

```diff
@@ -49,7 +49,8 @@ export type QuoteBridgeRequest = {
   WithBuyToken &
   WithQuoter &
   WithTrader &
-  TradeOptionalParameters
+  Omit<TradeOptionalParameters, 'slippageBps'> &
+  Partial<{ swapSlippageBps: latestAppData.SlippageBips; bridgeSlippageBps: latestAppData.SlippageBips }>
```

Comment:
Nitpick: would it be worth to create a type for this to simplify the expression. Something like: `WithBridgeSlippage` or similar.